libs/sdk-ui-kit/src/Icon/icons/Invite.tsx

```diff
@@ -0,0 +1,28 @@
+            <path
+                d="M4.03372 7.15451C4.17366 7.28279 4.3136 7.41107 4.48853 7.53935C4.54684 7.58599 4.61681 7.60932 4.69844 7.60932C4.80339 7.60932 4.90835 7.56267 4.97832 7.46937C5.09494 7.31777 5.05995 7.0962 4.89669 6.97958C4.75675 6.87462 4.62847 6.78133 4.52351 6.66471C4.20864 6.34984 3.95208 5.97666 3.77716 5.55684C3.59057 5.13701 3.50893 4.67054 3.50893 4.19241C3.50893 3.71427 3.60223 3.25946 3.77716 2.83964C3.96374 2.41981 4.20864 2.03497 4.52351 1.73176C4.85004 1.4169 5.22322 1.16034 5.64304 0.973747C6.47103 0.612231 7.5206 0.612231 8.34858 0.973747C8.76841 1.16034 9.15325 1.4169 9.46812 1.73176C9.78299 2.04663 10.0395 2.41981 10.2145 2.83964C10.4011 3.25946 10.4827 3.71427 10.4827 4.19241C10.4827 4.39066 10.6343 4.54226 10.8325 4.54226C11.0308 4.54226 11.1824 4.39066 11.1824 4.19241C11.1824 3.62098 11.0774 3.07287 10.8559 2.55975C10.6343 2.04663 10.3311 1.60349 9.95791 1.23031C9.57307 0.857129 9.12993 0.553922 8.62847 0.332348C7.61389 -0.110801 6.36608 -0.110801 5.36316 0.332348C4.8617 0.553922 4.41856 0.857129 4.03372 1.23031C3.64888 1.60349 3.34567 2.0583 3.13576 2.55975C2.91418 3.07287 2.80923 3.62098 2.80923 4.18074C2.80923 4.74051 2.91418 5.31194 3.13576 5.82506C3.35733 6.33818 3.66054 6.78133 4.03372 7.15451ZM9.80631 5.58016C7.48561 5.58016 5.60806 7.45771 5.60806 9.77841C5.60806 12.0991 7.48561 13.9767 9.80631 13.9767C12.127 13.9767 14.0046 12.0991 14.0046 9.77841C14.0046 7.45771 12.127 5.58016 9.80631 5.58016ZM9.80631 13.2886C7.87045 13.2886 6.30777 11.7143 6.30777 9.79007C6.30777 7.86588 7.88211 6.29153 9.80631 6.29153C11.7305 6.29153 13.3049 7.86588 13.3049 9.79007C13.3049 11.7143 11.7305 13.2886 9.80631 13.2886ZM11.9521 9.47521H10.1562V7.67929C10.1562 7.48104 10.0046 7.32943 9.80631 7.32943C9.60806 7.32943 9.45646 7.48104 9.45646 7.67929V9.47521H7.66054C7.46229 9.47521 7.31068 9.62681 7.31068 9.82506C7.31068 10.0233 7.46229 10.1749 7.66054 10.1749H9.45646V11.9708C9.45646 12.1691 9.60806 12.3207 9.80631 12.3207C10.0046 12.3207 10.1562 12.1691 10.1562 11.9708V10.1749H11.9521C12.1503 10.1749 12.3019 10.0233 12.3019 9.82506C12.3019 9.62681 12.1503 9.47521 11.9521 9.47521ZM1.04829 13.277C0.943337 13.277 0.873366 13.2536 0.791733 13.172C0.710101 13.0904 0.698439 12.7988 0.791733 12.449C0.89669 12.0874 1.05995 11.8076 1.42147 11.4227C1.63138 11.1895 1.89961 10.9912 2.19115 10.8047C2.50602 10.6181 2.85587 10.4431 3.24071 10.3149C3.49727 10.2216 3.80048 10.1516 4.13867 10.0816C4.32526 10.0466 4.45354 9.86004 4.40689 9.67346C4.37191 9.48687 4.19698 9.37025 3.98707 9.40523C3.61389 9.48687 3.29902 9.5685 3.00748 9.67346C2.57599 9.82506 2.16783 10.0116 1.81797 10.2332C1.46812 10.4548 1.15325 10.6997 0.89669 10.9679C0.453541 11.4344 0.231967 11.8076 0.0920249 12.2857C0.0103922 12.5772 -0.129549 13.3003 0.266952 13.6968C0.476865 13.895 0.721762 14 1.01331 14H5.56141C5.75966 14 5.91127 13.8484 5.91127 13.6501C5.91127 13.4519 5.75966 13.277 5.56141 13.277L1.04829 13.277Z"
+                fill={color ?? "#FFFFFF"}
```

Comment:
Should we use `rgba(255, 255, 255, 0.8)` as the default value, to be conform with the color in the mockup https://www.figma.com/file/YNqrdSzbRNy7Yd7WKeYI1p/22Q3-%2F-Invite-administrator?node-id=239%3A3997?

Reply:
The other icons and elements on the header use solid white. It made sense to me to use the white here as well to have it the same. Otherwise I would have to update other icons and texts to that transparent white and also use it for tab colors for applications (dashboards, analyze, data, etc.), they are solid white as well imho.

Reply:
Changed to `rgba(255, 255, 255, 0.8)` 👍